pkgs/development/misc/resholve/oildev.nix

```diff
@@ -13,6 +13,7 @@
 , glibcLocales
 , six
 , typing
+, yajl
```

Comment:
```suggestion
```

Reply:
I should really stop using `git commit -a`...